libgul14.spec

```diff
@@ -99,6 +99,9 @@ Requires:       %{name}-devel%{?_isa} = %{version}-%{release}
 %{_libdir}/%{name}.a
 
 %changelog
+* Fri Feb 16 2024 Soeren Grunewald <soeren.grunewald@desy.de> - 2.10-1
+- New upstream release
+
```

Comment:
For debian we invented a way to get rid of these manual changelog entries and stuff, but redhat brings that back? Not very nice. And this looks more like it belongs into a packaging repo, that fetched upstream and packages that.

Which is maybe the way to go anyhow, to not have the distribution stuff inside the (this) repo and have instead packager's repos. This is what I know and like from Arch. Well, probably all distribution releasers do it that way.

Reply:
Right, ideally the packaging should be a separate repository/project.